src/pages/DetailPage/Detail.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import './Detail.scss';
+import DishData from './Components/DishData';
+import TitleData from './Components/TitleData';
+
+class Detail extends React.Component {
+  render() {
+    return (
+      <main>
+        <div className="detailWrap">
+          <div className="topInfo">
+            <div className="imgDetailWrap">
+              <img
+                src="../images/DetailPage/pasta_above.jpeg"
```

Comment:
이미지는 public/images 폴더 하위에 넣어서 사용해주세요!

```suggestion
                src="/images/DetailPage/pasta_above.jpeg"
```

Reply:
이부분 반영해주세요~